src/server/frontend_wayland/wl_pointer.cpp

```diff
@@ -262,13 +264,43 @@ void mf::WlPointer::axis(MirPointerEvent const* event)
         needs_frame = true;
     }
 
-    if (h_scroll_discrete && version_supports_axis_discrete())
+    if (h_scroll_value120 && version_supports_axis_value120())
+    {
+        send_axis_value120_event(Axis::horizontal_scroll, h_scroll_value120);
+        needs_frame=true;
+    }
+    else if (h_scroll_discrete && version_supports_axis_value120())
+    {
+        auto value120 = h_scroll_discrete * 120;
+        // "The value120 must not be zero"
+        if (value120 != 0) 
```

Comment:
If `h_scroll_discrete != 0` (as checked in the else if above) than can't we assume `h_scroll_discrete * 120 != 0`?